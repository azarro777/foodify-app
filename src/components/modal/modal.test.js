import React from 'react';
import Modal from './modal';
import { shallow } from 'enzyme';

describe('Testing <Modal/> component', () => {
  const modal = shallow(<Modal />);
  it('Modal have rendered correctly', () => {
    expect(modal).toMatchSnapshot();
  });
  it('Modal state "inputs" is empty object', () => {
    expect((modal.inputs = {})).toBeObject();
  });
  it('testing handleChange function', () => {
    expect((modal.handleChange = () => {})).toBeFunction();
  });

  it('testing handleSubmit function', () => {
    expect((modal.handleSubmit = () => {})).toBeFunction();
  });

  it('testing getRundomId function', () => {
    expect((modal.getRundomId = () => {})).toBeFunction();
  });
});
