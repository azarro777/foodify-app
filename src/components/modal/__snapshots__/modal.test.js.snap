// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <Modal/> component Modal have rendered correctly 1`] = `ShallowWrapper {}`;
